docs/source/reference/glossary.rst

```diff
@@ -1,7 +1,73 @@
 Glossary
 ========
 
+
+API
```

Comment:
I've added definitions for all acronyms I find, but if you think some of these are too basic that any user of this software would already know, feel free to remove them.